docs/advanced-topics/best-practices/best-practices-dockstore.rst

```diff
@@ -0,0 +1,107 @@
+Whichever repository service you use, we recommend that your tool and workflow repositories are put under a non-personal organization, if possible.
+This allows for better organization and collaboration, and also provides a fallback for others if you become inactive on the git repository site.
+
+We also highly recommend that you do NOT host different tools and workflows on a single repo by putting them on different branches.
+Our reasoning is that branches are supposed to be used when working on a new feature or bug fix. That means they should eventually be merged with the repository's main branch.
+Another reason why we are against this practice is because it can lead to a lot of clutter on the Dockstore entries themselves.
+Let's say you have three workflows in a single repo on separate branches and there are many tags/releases related to each one.
+Depending on how you register the workflows on Dockstore, each entry could have the other two branches and every single tag/release
+on its Versions tab. All of this clutter can be overwhelming or confusing for users to see.
+
+Likewise, we don't recommend using even a single branch to hosts multiple entries. A repository should essentially represent one project, but that practice can be a grey area.
+If a lot of different workflows or tools work together or are closely related, it can be argued that they're all part of a single project.
+If that is the case, and you want to keep them tied together this way, make sure to give the descriptor files helpful file names.
+It can also help to keep them organized using different, well-named folders for each entry.
+
+One exception to this rule is when you describe the same tool or workflow with multiple descriptor languages. For example, we describe the same bamstats tool
+in our :doc:`Getting Started with CWL <../../getting-started/getting-started-with-cwl>` and :doc:`Getting Started with WDL <../../getting-started/getting-started-with-wdl>` tutorials.
+A single tool on Dockstore can have CWL and WDL descriptor files. Users will be able to see you've provided both and can easily switch between the descriptors in the UI.
+Unfortunately, you cannot have a single workflow entry on Dockstore contain multiple descriptor types. You will have to register them as separate workflows.
+
+
+Improving Your Entries On Dockstore
+-----------------------------------
+
+Making It Easier to Find
+^^^^^^^^^^^^^^^^^^^^^^^^
+There are a few things you can do to make your tool/workflow easier to find on our site.
+
+Let's start with making your entry easier to find on our `search <https://dockstore.org/search>`_ page. One way is to add author and description metadata to your descriptor file.
+Adding an author will make it selectable on the Author facet and a description helps because doing a text search sifts through the text of your descriptor files.
```

Comment:
How does this interact with the README.md? Are the contents of the README.md indexed?

Should you cover using the README.md to document things as an alternative?

Reply:
Just checked. I updated a workflow README to have a word that didn't produce results. Once it was refreshed, it came up by searching for that word